task/04-arrays-tasks.js

```diff
@@ -454,7 +508,15 @@ function getIdentityMatrix(n) {
  *     3, 3   => [ 3 ]
  */
 function getIntervalArray(start, end) {
-   throw new Error('Not implemented');
+   /*
+   let arr = []
+   for(let i = start; i <= end; i++)
+      arr.push(i)
+
+   return arr
+   */
+
+   return Array.from({length: end - start + 1},(value, index)=> start + index)
```

Comment:
```suggestion
   return Array.from({ length: end - start + 1 }, (_, index) => start + index)
```

Don't forget about the consistency in spaces, it also gives a bit cleaner look to the code.
And, in such cases, where we need only the second argument of the arrow function, we can declare the first one as `_` (underscore)

Reply:
So the arguments we don't need can always be left as _? Originally, I wanted to come up with a solution without using unused arguments, but the solution would have been more complicated

Reply:
Yep, you are right